frontend/utils/stringUtils.ts

```diff
@@ -13,3 +13,13 @@ export const plural = (value: number, phrase: string) => {
 export const isValidSemver = (semverInput: string) => {
   return !!semver.valid(semverInput)
 }
+
+export const isPortNumber = (portNumber: string) => {
```

Comment:
Very minor, but this name makes it look like a type guard. Maybe change it to clarify that it's a validation check? `isValidPortNumber`?